### PR TITLE
V abt/tab coversations

### PR DESCRIPTION
**Pull request by AbhijeetThakur-msft**
The sample is to Create conversational tabs with the following actions:

- Create a new conversation.
- Close conversation view.
- Continue an existing conversation.
- Deeplink to redirect to a conversation.

### CODE REVIEW

File: samples/tab-conversations/nodejs/server/static/styles/custom.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+html, body, div.surface, div.panel {
+    height: 100%;
+    margin: 0;
+}
+
+div.panel {
+    padding: 15px;
+}
+table {
```
Comment (v-chetsh) [inline samples/tab-conversations/nodejs/server/static/styles/custom.css:9]: One line space before it.

File: samples/tab-conversations/nodejs/server/views/configure.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                entityId: createTabUrl(),
+            });
+
+            // Tells Microsoft Teams platform that we are done saving our settings. Microsoft Teams waits
+            // for the app to call this API before it dismisses the dialog. If the wait times out, you will
+            // see an error indicating that the configuration settings could not be saved.
+            saveEvent.notifySuccess();
+        });
+
+        function choiceOnChange(element) {
```
Comment (v-chetsh) [inline samples/tab-conversations/nodejs/server/views/configure.html:31]: Kindly add function summary

File: samples/tab-conversations/nodejs/server/views/configure.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                // This API tells Microsoft Teams to enable the 'Save' button. Since Microsoft Teams always assumes
+                // an initial invalid state, without this call the 'Save' button will never be enabled.
+                microsoftTeams.settings.setValidityState(
+                    selectedTab === 'conversationTab' || selectedTab === 'conversationTab'
+                );
+            }
+        }
+
+        // Create the URL that Microsoft Teams will load in the tab. You can compose any URL even with query strings.
+    function createTabUrl() {
```
Comment (v-chetsh) [inline samples/tab-conversations/nodejs/server/views/configure.html:45]: Adjust alignment

File: samples/tab-conversations/nodejs/Readme.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
+# Conversational Tabs
```
Comment (v-chetsh) [inline samples/tab-conversations/nodejs/Readme.md:1]: Kindly link your Application page with main Readme.md file.

**Review by v-chetsh — APPROVED** [review]
Approving the PR with suggestions & as per internal discussion.

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**